ts/Series/Treemap/TreemapNode.ts

```diff
@@ -0,0 +1,74 @@
+/* *
+ *
+ *  (c) 2010-2022 Pawel Lysy
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ * Imports
+ *
+ * */
+
+import type TreemapSeries from './TreemapSeries.js';
+import type BBoxObject from '../../Core/Renderer/BBoxObject.js';
+import type TreemapPoint from './TreemapPoint.js';
+
+interface TreemapNode {
```

Comment:
- Interface for class properties should follow the class
- Section comments missing